projects/fsastorefrontlib/src/cms-components/my-account/policy/policies/policies.component.ts

```diff
@@ -65,6 +67,10 @@ export class PoliciesComponent implements OnInit, OnDestroy {
     }
   }
 
+  setPolicyOriginStep() {
+    this.winRef.sessionStorage.setItem(ORIGIN_POLICIES_PAGE, 'true');
```

Comment:
Storing the information when user has landed on policy details via My Policies page.